index.md

```diff
@@ -179,7 +196,7 @@ PATCH /jsonapi/node/article/{id}?lang=fr
 404 Not Found
 
 // If an unstranslatable field value is specified:
-400 Bad Request
+422 Unprocessable entity
```

Comment:
I'm wondering whether this should apply also to all the other `400`s: according [to the spec](https://tools.ietf.org/html/rfc4918#section-11.2), this should be returned when the request is syntactically valid but semantically wrong. I believe this is the case for all the situations where we have a mismatch between the various ways to specify the content language.

Reply:
I'm fine with that interpretation 👍